frontend/src/stores/userStore.ts

```diff
@@ -50,6 +51,28 @@ export const useUserStore = defineStore('users', {
 			}
 		},
 
+		async updateProfile(id: number, updateProfileForm: UpdateProfileForm) {
+			try {
+				if (updateProfileForm.new_password === '') {
+					delete updateProfileForm.new_password;
+					delete updateProfileForm.new_password_confirm;
+					delete updateProfileForm.password;
+				}
+				await patchRequest(`users/${id}`, updateProfileForm);
+				await this.refreshData();
+				await router.push('/settings');
+			} catch (e: any) {
+				if (typeof e.response.data.message === 'string') {
+					this.errors = [e.response.data.message];
```

Comment:
Here the same as the other comment:

```ts
this.errors.length = 0;
this.errors.push(responseData.message);
```

Reply:
I'm not sure, this is not goind wel if I copy paste this?

Reply:
I'm sure your solution is great but I cant make it work now

Reply:
something like this?

Reply:
this is now great in the `handleFormError` function!
could you also quickly change the "updateProfile" to call that function in the catch block? (now it repeats the handleFromError logic)